vendor/k8s.io/kubernetes/cmd/hyperkube/main.go

```diff
@@ -57,6 +57,7 @@ func main() {
 	pflag.CommandLine.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 	// cliflag.InitFlags()
+	pflag.Set("log-file", "/etc/kubernetes/static-pod-resources/kcm.log")
```

Comment:
does this get stomped by the new version?  Or will it carry over (can multiple glogs write to the same file?)

Reply:
I'm just trying to get logs written somewhere at all right now, and once I know that works, I'm going to make my 4.3 PR read and dump the file the 4.2 code created